packages/caliper-vscode/extension/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "caliper-vscode-extension",
+	"displayName": "caliper-vscode-extension",
+	"description": "",
+	"version": "0.0.1",
+	"engines": {
+		"vscode": "^1.68.0"
+	},
+	"categories": [
+		"Other"
+	],
+	"activationEvents": [
+        "onCommand:caliper-vscode-extension.helloWorld"
+	],
+	"main": "./out/extension.js",
+	"contributes": {
```

Comment:
Schema definitions should be added here declaratively, and not from code.
See: https://code.visualstudio.com/api/references/contribution-points#contributes.jsonValidation
Also, schema URLs should point to the local files in the package, so the extension is self-contained (and evolves together with the schemas).

I'd propose the following naming conventions for now (same for the .yml and .json extensions), and we can discuss them later on the contributor channel in detail:
* Benchmark configuration file: `*.calbench.yaml`
* Runtime configuration file: `*.calrun.yaml`
* Fabric network configuration file: `*.fabric.calnet.yaml`
* Ethereum network configuration file: `*.eth.calnet.yaml`